portal/models/message.py

```diff
@@ -139,6 +151,7 @@ def send_message(self, cc_address=None):
         """
         message = Message(
             subject=self.subject,
+            extra_headers=extra_headers(),
```

Comment:
Will these headers only be added to reminder emails, or all emails from the system (eg flask-user emails)?
(I think only reminder emails, but want to be sure...)